docs/howtos/runtime_sessions.rst

```diff
@@ -40,5 +80,4 @@ large number of circuits that can't fit in a single job, it may be helpful to fo
     # This will prevent further jobs from being submitted without terminating current jobs
     backend.close_session()
 
-Note that runtime primitives are not currently supported natively in Qiskit Experiments, so  
-the ``backend.run()`` path is required to run experiments.
+
```

Comment:
Can we delete this whole session part? `backend.session` is also deprecated and the note about needing at least qiskit-ibm-runtime 0.15 feels pretty dated when 0.31 is out. It might be nice to keep some mention that circuits will be executed using the sampler's `run()` method that could be run inside of a session.